btcjson/chainsvrcmds.go

```diff
@@ -63,10 +63,15 @@ type CreateRawTransactionCmd struct {
 // NewCreateRawTransactionCmd returns a new instance which can be used to issue
 // a createrawtransaction JSON-RPC command.
 //
-// Amounts are in BTC.
+// Amounts are in BTC. Passing in nil and the empty slice is equivalent, both
```

Comment:
nit: It seems the remark about "passing nil" applies to `amounts`, instead of `inputs`.

Reply:
Fixed